website/docs/cloud-docs/overview/activate-flex.mdx

```diff
@@ -63,6 +63,8 @@ Navigate to the **HCP Org > Billing**.  If HashiCorp sales has allocated Flex, t
 
 Navigate to your Terraform Cloud organization setting's **Plan & Billing** page. If your Terraform Cloud user account is not managed by HCP, you will be prompted to log in to an HCP account that has the `admin` role in the HCP organization with HashiCorp Flex allocated. You must have admin privileges in the HCP organization to manage the Terraform Cloud organization's billing plan.  
 
+![Prompt for user to link their TFC account to an HCP account](/img/docs/tfc-plan-and-billing.png)
```

Comment:
can we just annotate this image to have a box around "Continue with HCP" account instead of the arrows and text? feels like it's repeating the link text and button name